src/stancjs/stancjs.ml

```diff
@@ -5,8 +5,28 @@ open Analysis_and_optimization
 open Middle
 open Js_of_ocaml
 
-let stan2cpp model_name model_string =
+let version = "%%NAME%% %%VERSION%%"
+
+let warn_uninitialized_msgs
+    (uninit_vars : (Location_span.t * string) Set.Poly.t) =
+  let show_var_info (span, var_name) =
+    Location_span.to_string span
+    ^ ":\n" ^ "  Warning: The variable '" ^ var_name
+    ^ "' may not have been initialized.\n"
+  in
+  let filtered_uninit_vars =
+    Set.filter ~f:(fun (span, _) -> span <> Location_span.empty) uninit_vars
+  in
+  Set.Poly.to_list
+    (Set.Poly.map filtered_uninit_vars ~f:(fun v_info -> show_var_info v_info))
+
+let stan2cpp model_name model_string flags_string =
+  let flags = String.split_on_chars ~on:[' '] flags_string in
```

Comment:
For now, the arguments are a string with space-delimited flags. It would be more Javascriptish if it were an array of string. Are "--" necessary in that case?

Reply:
I don't think they're necessary. I will push an attempt at the array version but I can't get the tests to run locally - any tips?

Reply:
(I got the tests to run!)

But not sure if we need to include the "--". I can see arguments either way. Happy with whatever you decide here.